src/modules/profile/cqrs/command/upload-profile-image.command.ts

```diff
@@ -13,14 +13,16 @@ export interface UploadProfileImageCommandArgs {
 @Injectable()
 export class UploadProfileImageCommand implements ICommand {
     constructor(
-        @Inject(S3ClientService) private readonly s3ClientService: S3ClientService,
+        @Inject(BlobClientService) private readonly blobClientService: BlobClientService,
         private readonly profileRepository: IProfileRepository,
     ) {}
 
     // TODO: Implement validations and errors
     async execute(args: UploadProfileImageCommandArgs): Promise<void> {
-        const avatar = await this.s3ClientService.upload(args.file, args.id, UploadImageTypeEnum.PROFILE);
+        const avatar = await this.blobClientService.upload(args.file, args.id, UploadImageTypeEnum.PROFILE);
         const profile = await this.profileRepository.findOne({ userId: args.id });
+        // TODO: Not sure how to get an id here? Previously with S3 we could just get the Key but I don't think a similar value
+        // exists in BlockBlobUploadResponse? I just return the file path now
```

Comment:
Should be fine, the returned `Key` from S3 was basically the relative location, similar to the one you return now.